recipes-ni/base-files-nilrt/base-files-nilrt_1.0.bb

```diff
@@ -38,6 +38,12 @@ DEPENDS += "niacctbase"
 do_install () {
 	install -d -m 0755 -o ${LVRT_USER} -g ${LVRT_GROUP} ${D}${sysconfdir}/natinst/share/
 
+	# Something in base is installing /etc/natinst/share/uninstall that may not be
+	# getting installed for SystemLink. nisid will fail if this directory doesn't
+	# install, so this component ought to install /etc/natinst/share/uninstall
+	# itself, regardless of if some other component ultimately will too.
+	install -d -m 0755 ${D}${sysconfdir}/natinst/share/uninstall
```

Comment:
Can we file a bug to NISID to create this directory, if they depend on it?

What's supposed to go in that directory? Uninstall hooks for NI packages?

Reply:
From what I can see, the contents of that directory are postrm rules for the ni_systemlink CDFs that used to be installed as dependencies of the System Image. We have since moved all the IPKs installed by the CDFs to the base image. There doesn't seem to be anything else in that directory. I think it is best we file a bug to nisid or work with them to figure out what else goes in this directory and if it is still relevant in RAUC. Here's a snapshot of the directory contents:
![image](https://user-images.githubusercontent.com/62362030/107997708-42b80380-6f98-11eb-8f56-0c63a9476884.png)

Reply:
I don't have much to add, except that I think we've been told that NISID would be gone in RAUC images.

Reply:
Ok. It looks like we don't need this dir so I'm removing this commit for now. We can add back later if thats not the case.